syntax = "proto3";

package greet;


service Greeter {

	rpc Give_Weather_Details (WeatherRequest) returns (WeatherResponse);



}


message WeatherRequest {
string city_name = 1;
}
message WeatherResponse {
string city_name = 1;
float temperature = 2;
string description = 3;
int32 humidity = 4;
float wind_speed = 5;
string data_created=6;
}
